File: CH3-desicion-sructure-and-boolean-logic/EX4-magic-dates/source-code/magicDates.java
import java.util.Scanner;

public class magicDates {

  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    int month;
    int day;
    int year;
    int result;

    System.out.print("Enter a month :");
    month = input.nextInt();
    System.out.print("Enter a day   :");
    day = input.nextInt();
    System.out.print("Enter a year  :");
    year = input.nextInt();

    result = month * day;

    if (result == year) {
      System.out.println("the date is magic");
    } else {
      System.out.println("the date is not magic");
    }

    input.close();
  }

}

File: CH3-desicion-sructure-and-boolean-logic/EX4-magic-dates/magicDates.js
const input = require("prompt-sync")();

let month;
let day;
let year;
let result;

month = input("Enter a month :");
day = input("Enter a day   :");
year = input("Enter a year  :");

result = month * year;

if (result == year) {
  console.log("the date is magic");
} else {
  console.log("the date is not magic");
}
